test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/ClusteredWorkbenchKieServerPersistentScenarioLdapIntegrationTest.java

```diff
@@ -74,29 +82,29 @@ public static void cleanEnvironment() {
 
     @Test
     @Category(JBPMOnly.class)
-    public void testProcessFromExternalMavenRepo() {
-        ProcessTestProvider.testDeployFromKieServerAndExecuteProcesses(deploymentScenario.getKieServerDeployment());
+    public void testProcessFromInternalMavenRepo() {
+        processTestProvider.testDeployFromKieServerAndExecuteProcesses(deploymentScenario.getKieServerDeployment());
```

Comment:
Just looking at this name, change and thinking about naming. Under internal maven repo I see maven repository running in Busines Central, but if am I right here is used internal maven repo for Nexus runnnig in the same project right? Wouldn't be better to just use there maven repo (without external or internal)?

Reply:
We could use directly 'FromMavenRepo' indeed